### PR TITLE
[Team 5 Agent] bug fix and updated config

### PR DESCRIPTION
**Pull request by kyjustinma**
# Summary

-  Fixed bug where opinions would be deducted more than once in the same turn in foraging
- Tidied up c.LogF outputs 
- Changed some parameters for opinions because of the bug

## Additional Information

Nothing really

## Test Plan

?       github.com/SOMAS2020/SOMAS2020  [no test files]
ok      github.com/SOMAS2020/SOMAS2020/internal/clients/team1   (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/clients/team2   (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/clients/team3   (cached)
?       github.com/SOMAS2020/SOMAS2020/internal/clients/team3/adv       [no test files]
ok      github.com/SOMAS2020/SOMAS2020/internal/clients/team3/dynamics  (cached)
?       github.com/SOMAS2020/SOMAS2020/internal/clients/team3/ruleevaluation   [no test files]
ok      github.com/SOMAS2020/SOMAS2020/internal/clients/team4   (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/clients/team5   (cached)
ok      github.com/SOMAS2020/SOMAS2020/interal/hewontfindme5  (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/clients/team6   (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/common/baseclient       (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/common/config   (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/common/disasters        (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/common/foraging (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/common/gamestate        (cached)
?       github.com/SOMAS2020/SOMAS2020/internal/common/roles    [no test files]
ok      github.com/SOMAS2020/SOMAS2020/internal/common/rules    (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/common/shared   (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/common/simulation       (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/common/voting   (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/server  (cached)
ok      github.com/SOMAS2020/SOMAS2020/internal/server/iigointernal     (cached)
ok      github.com/SOMAS2020/SOMAS2020/pkg/fileutils    (cached)
?       github.com/SOMAS2020/SOMAS2020/pkg/gitinfo      [no test files]
?       github.com/SOMAS2020/SOMAS2020/pkg/logger       [no test files]
?       github.com/SOMAS2020/SOMAS2020/pkg/miscutils    [no test files]
?       github.com/SOMAS2020/SOMAS2020/pkg/sysutils     [no test files]
?       github.com/SOMAS2020/SOMAS2020/pkg/testutils    [no test files]

### CODE REVIEW

File: internal/clients/team5/forage.go [hunk 1]
```diff
@@ -342,22 +347,15 @@ func (c *client) MakeForageInfo() shared.ForageShareInfo {
 	if c.getTurn() > c.config.InitialForageTurns { // for the turns we are NOT doing initial forage
 		for _, FOutcome := range c.forageHistory {
 			for _, returns := range FOutcome {
-				for _, team := range c.getAliveTeams(false) { // For all alive teams
-					if c.getTurn() > c.config.DeerTurnsToLookBack { // Turns greater than look back
-						if returns.turn > c.getTurn()-c.config.DeerTurnsToLookBack &&
-							returns.turn <= c.getTurn() &&
-							returns.team == team { // If a certain team within a certain range
+				if c.getTurn() > c.config.DeerTurnsToLookBack && // prevent looking at negative turns
+					c.getTurn() >= c.getTurn()-c.config.DeerTurnsToLookBack { // Turns greater than look back
```
Comment (NGovani) [inline internal/clients/team5/forage.go]: shouldn't the RHS be FOutcome.turn

**Review by NGovani — APPROVED** [review]
one small thing but im preapproving since its not really game breaking

File: internal/clients/team5/forage.go (hunk 1 above)
Comment (kyjustinma) [reply internal/clients/team5/forage.go]: fixed, thanks for the spot. Could you also merge for me? I have no write acccess

File: internal/clients/team5/forage.go [hunk 2]
```diff
@@ -407,3 +406,22 @@ func (c *client) MakeForageInfo() shared.ForageShareInfo {
 	c.Logf("[MakeForageInfo][%v]: %+v", c.getTurn(), forageInfo)
 	return forageInfo
 }
+
+// difference between 2 slices (yes I ripped it off online)
+func difference(slice1 []shared.ClientID, slice2 []shared.ClientID) []shared.ClientID {
```
Comment (lhl2617) [inline internal/clients/team5/forage.go:411]: (you may ignore)

`REEEEEEEEEEEEEEEEEEEEEEEEEEE O(n^2)`

**Pull request merged**